packages/design-system/src/components/N8nBadge/__tests__/__snapshots__/Badge.spec.ts.snap

```diff
@@ -0,0 +1,5 @@
+// Vitest Snapshot v1
+
+exports[`components > N8nBadge > props > should render default theme correctly 1`] = `"<span class=\\"_default_13dw2_9 _badge_13dw2_1\\"><span class=\\"_size-small_9dlpz_24 _bold_9dlpz_1\\" style=\\"line-height: 1;\\">Default badge</span></span>"`;
+
+exports[`components > N8nBadge > props > should render secondary theme correctly 1`] = `"<span class=\\"_secondary_13dw2_16 _badge_13dw2_1\\"><span class=\\"_size-medium_9dlpz_19 _regular_9dlpz_5\\" style=\\"line-height: 1;\\">Secondary badge</span></span>"`;
```

Comment:
we should mock out `n8n-text` instead of rendering it completely

Reply:
Agree.